app/models/post.rb

```diff
@@ -1,5 +1,8 @@
 class Post < ApplicationRecord
-  validates :content, presence: true
+  validates :content,
+            :user_id, presence: true
   validates :grass_tags,
             :only_followers, presence: false
+
+  belongs_to :user
 end
```

Comment:
This is 1/2 of the major change.  Post now has a User ID as a foreign key, and the UserPosts joins table has been repurposed to be a UserLike table, for future implementation